apis/core.oam.dev/v1beta1/appdeployment_types.go

```diff
@@ -178,7 +178,7 @@ type AppDeploymentStatus struct {
 
 // AppDeployment is the Schema for the AppDeployment API
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:categories={oam},shortName=appdeploy
+// +kubebuilder:resource:categories={oam},shortName=appdeploy;appdpy
```

Comment:
appdeloy is enought for short names, `appdpy` is not necessary

Reply:
Done